doc/source/advanced.md

```diff
@@ -1,7 +1,14 @@
 # Advanced Topics
 
 This page contains a grab bag of various useful topics that don't have an easy
-home elsewhere. Most people setting up JupyterHubs on popular public clouds
+home elsewhere:
+
+- Ingress
```

Comment:
Here could we just use:

```
.. contents::
   :local:
   :depth: 1
```
?

Reply:
(that way it'd auto-update and also add links to the sections)

Reply:
This is recommonmark, not sure it supports it?

Reply:
@choldgraf I couldn't get this to work using recommonmark. If you can, please do :-)

Reply:
I just gave it a shot and couldn't get it to work either. I have a feeling it's a bug in recommonmark :-/

Reply:
I did some investigation the other day for the JupyterLab folks re: recommonmark and it hasn't had a release since last May. There are some issues with compatibility and other things. We may need to fix recommonmark :( at some point or look at a docutils/sphinx solution. For now, it seems fine if we don't get fancy. @choldraf :(